app/models/concerns/form526_workflow.rb

```diff
@@ -0,0 +1,173 @@
+# frozen_string_literal: true
+
+module Form526Workflow
+  extend ActiveSupport::Concern
+
+  FORM_526 = 'form526'
+  FORM_526_UPLOADS = 'form526_uploads'
+  FORM_4142 = 'form4142'
+  FORM_0781 = 'form0781'
+  FORM_8940 = 'form8940'
+  FLASHES = 'flashes'
+  SUBMIT_FORM_526_JOB_CLASSES = %w[SubmitForm526AllClaim SubmitForm526].freeze
+
+  # Kicks off a 526 submit workflow batch. The first step in a submission workflow is to submit
+  # an increase only or all claims form. Once the first job succeeds the batch will callback and run
+  # one (cleanup job) or more ancillary jobs such as uploading supporting evidence or submitting ancillary forms.
+  #
+  # @return [String] the job id of the first job in the batch, i.e the 526 submit job
+  #
+  def start_evss_submission(_status, options)
+    submission = Form526Submission.find(options['submission_id'])
+    id = submission.id
+    workflow_batch = Sidekiq::Batch.new
+    workflow_batch.on(
+      :success,
+      'BatchDoneHandlers#perform_ancillary_jobs_handler',
```

Comment:
This might need to change to
```suggestion
      'Form526Workflow::BatchDoneHandlers#perform_ancillary_jobs_handler',
```
but I'm not sure how to test it because "Batches do not support the inline or fake testing modes" -https://github.com/mperham/sidekiq/wiki/Batches#testing.

I might create callback classes instead: https://github.com/mperham/sidekiq/wiki/Batches#callbacks